dependencies/src/main/java/com/google/cloud/tools/opensource/classpath/LinkageCheckerArguments.java

```diff
@@ -200,7 +202,13 @@ private static Options configureOptions() {
 
     if (commandLine.hasOption("b") || commandLine.hasOption("a")) {
       List<Artifact> artifacts = getArtifacts();
-      cachedInputClasspath = classPathBuilder.resolve(artifacts).getClassPath();
```

Comment:
This method "resolve" does not throw RepositoryException any more.

Handling the unresolved artifact error by checking `result.getArtifactProblems()`.